packages/jaeger-ui/src/components/QualityMetrics/ScoreCard.test.js

```diff
@@ -24,47 +23,44 @@ describe('ScoreCard', () => {
   const max = 108;
 
   it('renders as expected when score is below max', () => {
-    expect(
-      shallow(
-        <ScoreCard
-          link={link}
-          score={{
-            label,
-            max,
-            value,
-          }}
-        />
-      )
-    ).toMatchSnapshot();
+    const { container } = render(
```

Comment:
couldn't you just change shallow->render and keep the rest the same, to minimize the changes?

Reply:
As I have discovered till now, the render function is not a direct replacement for shallow. As per docs, 
> Enzyme's shallow renderer doesn't render sub-components, so React Testing Library's render method is more similar to Enzyme's mount method.

This is also the reason why snapshot file is modified with a lot of code additions.

Reply:
It can be inlined, but we need to use container out of render function to match the snapshot, as it doesn't provide that utility itself.